.github/workflows/release.yaml

```diff
@@ -51,9 +51,9 @@ jobs:
           prerelease: false
 
       - name: Upload the artifacts
-        uses: skx/github-action-publish-binaries@release-1.3
+        uses: skx/github-action-publish-binaries@master
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           releaseId: ${{ needs.create_release.outputs.id }}
-          args: 'charts-aux-bundled.tgz'
+          args: 'Bundle-Charts-Release-${{ env.TAG_NAME }}.tgz'
```

Comment:
Do we need to use the fullpath here?
`graphistry-helm/bundle-charts/Bundle-Charts-Release-${{ env.TAG_NAME }}.tgz`

Reply:
this just uploads release to the release page it doenst store it in repo i dont thinl

Reply:
I see, do you know where the task `Run chart-releaser` or the task `Create Release` outputs the final tarball? maybe they use some default directory or a temporary one¿?

Reply:
I think they use a temp dir , the action repository doesn't say, will have to dig into the code, Leo recommended this one so that's what I went with

Reply:
chart releaser has been used since the inception of the repo - https://github.com/helm/chart-releaser-action its nice it creates a index.yaml and deploys to github pages and makes a self contained repo and also packages the helm charts, similar to our bundle script but its only pointed at a specific directory and only includes main charts

Reply:
> I think they use a temp dir 

Got it, yes if we use a temp dir then I guess this line:
`args: 'Bundle-Charts-Release-${{ env.TAG_NAME }}.tgz'`
will do its stuff under the same workdir right?
I want to double check that part here.

Reply:
Yea I think so here: https://github.com/skx/github-action-publish-binaries/blob/44887b225ceca96efd8a912d39c09ad70312af31/upload-script#L134

Reply:
Perfect!